packages/traceability/issuer-api/src/pods/blockchain/blockchain-properties.entity.ts

```diff
@@ -26,10 +26,15 @@ export class BlockchainProperties extends ExtendedBaseEntity {
     platformOperatorPrivateKey: string;
 
     @Column({ nullable: true })
-    rpcNodeFallback: string;
+    rpcNodeFallback?: string;
+
+    @Column({ nullable: true })
+    privateIssuer?: string;
 
     wrap(signerOrPrivateKey?: Signer | string): IBlockchainProperties {
-        const web3 = getProviderWithFallback(this.rpcNode, this.rpcNodeFallback);
+        const web3 = getProviderWithFallback(
+            ...[this.rpcNode, this.rpcNodeFallback].filter((url) => !!url)
```

Comment:
Just a super NIT, but for that kind of functions it's nice to have a helper. lodash has `compact`.

For this case though maybe you consider `.filter(Boolean)` since it better explains intention? It's just my opinion though.